argocd/tests/test_argocd.py

```diff
@@ -0,0 +1,85 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.argocd import ArgocdCheck
+from datadog_checks.base.constants import ServiceCheck
+
+from . import common
+from .utils import get_fixture_path
```

Comment:
The same question as with the `metrics` module: is there a convention of putting that function into a separate module? If not, how does this compare to 

a) providing it as a fixture in case you want to share it with `test_e2e.py`
b) defining it here, closer to where it is used

Reply:
bump ;)